conf/etc/nginx/sites-available/default

```diff
@@ -23,6 +23,7 @@ server {
 
 	root /opt/piecewise_web;
 	index index.html index.htm;
+	sendfile off;
```

Comment:
I removed this added line and tested on an Ubuntu 16.04 LTS host system. It had no effect on my test VM deployment, so I'll chock this up to testing related to an issue with my previous system, which likely had an incorrectly configured Virtualbox install. I had uninstalled and reinstalled Virtualbox packages many times, so that system was a bit out of standard configuration.

This line is not needed and has been removed in my latest commit.